### PR TITLE
771-Unable change assigned to unassigned in backlog

### PR DESCRIPTION
**Pull request by kanjilal1122**

### CODE REVIEW

**Comment by dipanjanpanja6** [comment]
Please add issue ID in the description or in branch name

File: src/pages/projectBacklog/components/SprintIssue.jsx [hunk 1]
```diff
@@ -204,17 +204,41 @@ export default function SprintIssue({
         <Typography variant="body2" sx={{ padding: "6px 12px", color: "#64748b" }}>
           Change assignee to:{" "}
         </Typography>
+        {/* <MenuItem value="" >
```
Comment (dipanjanpanja6) [inline src/pages/projectBacklog/components/SprintIssue.jsx:207]: remove this commented code

Comment (dipanjanpanja6) [reply src/pages/projectBacklog/components/SprintIssue.jsx:207]: format code before push.
https://code.visualstudio.com/docs/editor/codebasics#:~:text=VS%20Code%20has%20great%20support,)%20%2D%20Format%20the%20selected%20text.

**Review by dipanjanpanja6 — CHANGES_REQUESTED** [review]

**Comment by kanjilal1122** [comment]
Remove the commented code .

**Review by dipanjanpanja6 — APPROVED** [review]

**Pull request merged**